docs/pages/docs/index.mdx

```diff
@@ -9,22 +9,29 @@ import Features from "../../components/Features";
 
 <Features page="docs" detailed={false} />
 
-Turborepo is a blazing fast build system for JavaScript/TypeScript _monorepos_: codebases containing multiple
-projects, often using multiple frameworks, in a single, unified
-code repository.
+Turborepo is a blazing fast build system for JavaScript/TypeScript monorepos - codebases containing multiple projects in a single, unified repository.
 
-Monorepos have many advantages, but they require appropriate tooling to scale. Turborepo provides this tooling and more. It leverages advanced build system ideas and techniques to speed up development, but with a fraction of the configuration and complexity. Turborepo abstracts all the annoying configuration, scripts, and tooling into `turbo.json`, so you can focus on _what_ you're shipping instead of worrying about _how_ it'll get shipped.
+## The problem
 
-Most importantly, unlike other build systems, Turborepo is designed to be incrementally adopted, so you can gradually and partially add it to most codebases in a few minutes.
+Monorepos have many advantages - but **they struggle to scale**. Each repository you add has its own test suite, its own build process and its own pipeline. A single monorepo might have **hundreds of tasks to perform**.
 
-## Why Turborepo?
+## The solution
 
-- **Faster, incremental builds**: Building once is painful enough, Turborepo will remember what you (or someone on your team) built and skip work that's already been done.
-- **Content-aware hashing**: Turborepo looks at the contents of your files, not timestamps, to figure out what needs to be built.
-- **Cloud caching**: Share a cloud build cache with your teammates and CI/CD for even faster builds.
-- **Parallel execution**: Execute builds using every core at maximum parallelism without wasting idle CPUs.
-- **Zero runtime overhead**: Turborepo doesn't interfere with your runtime code or touch your sourcemaps. It does what it does and then gets out of your way.
-- **Task pipelines**: Define the relationships between your tasks and then let Turborepo optimize what to build and when.
-- **Pruned subsets**: Speed up PaaS deploys by generating a subset of your monorepo with only what's needed to build a specific target.
-- **Convention-based config**: Reduce complexity through convention. Fan out configuration with just a few lines of JSON.
-- **Profile in your browser**: Generate build profiles and view them in a browser to understand which tasks are taking the longest.
+**Turborepo speeds up your monorepo**. Our remote cache stores the result of all your tasks, meaning that **your CI never needs to do the same work twice**.
```

Comment:
```suggestion
**Turborepo speeds up the tasks in your monorepo**. Our remote cache stores the results of all your tasks, so that **your CI never needs to do the same work twice**.
```

Monorepos don't move themselves, so a tool can't speed them up 😛

Reply:
I disagree - monorepos can be slow or fast!

Reply:
We may have to agree to disagree 😉 . Monorepos (by your own definition above) are:

> codebases containing multiple projects in a single, unified repository.

How can a codebase be fast or slow? Code can _run_ fast or slow and builds can _execute_ fast or slow; a codebase has no transformational aspect that has a speed!

Not a hill I want to die on though, obviously 😄  .

Reply:
Agree with @mehulkar here. Turbo also doesn't speed up the tasks, it runs them more efficiently and intelligently which results in faster builds. It explicitly doesn't touch the task itself.

Reply:
@jaredpalmer's take means that @mehulkar's suggestion (speeds up the tasks in your monorepo) is also not quite right.

I am still happy with "Turborepo speeds up your monorepo" as the best phrase (we've found so far) to represent the problem that turbo is solving.

"Turbo optimizes your monorepo" isn't specific enough
"Turbo optimizes how your monorepo runs tasks" doesn't address the _emotion_ behind the problem statement. "Monorepos are so slow" is exactly what folks are saying, why not reply with "Turborepo speeds them up".

Reply:
How about "Turborepo makes your monorepo smarter" ?

Reply:
@mehulkar I've gone with `Turborepo solves your monorepo's scaling problem`